File: mario_more.py
from cs50 import get_int


def mario():
    # ask user about height in case when input is less than 0 or greater than 23 repeat until input will be proper
    while True:
        height = get_int("Height: ")
        if (height > -1) and (height < 24):
            break
    # variable rowLength store how many charachter are in each row
    row_length = height
    how_many_hashes = 1
    for row in range(height):
        # variable store how many space in the actual row we print on the output
        how_many_spaces = row_length - how_many_hashes
        # printing spaces
        for number in range(how_many_spaces):
            print(" ", end="")
        # printing hashes
        for number in range(how_many_hashes):
            print("#", end="")
        print("  ", end="")
        # printing hashes
        for number in range(how_many_hashes):
            print("#", end="")
        print()
        # increment number of hashes
        how_many_hashes += 1


mario()

File: mario_more.c
#include <stdio.h>
#include <stdlib.h>
#include <cs50.h>

int main(void)
{
    int height;


    do
    {
        height = get_int("Height: ");
    }
    while (height < 0 || height > 23);

    for (int i = 1; i <= height; i++)
    {

		for (int j = height - i; j > 0; j--)
    	{
    		printf(" ");
		}

		for (int k = i; k > 0; k--)
		{
			printf("#");
		}

		printf("  ");

		for (int k = i; k > 0; k--)
		{
			printf("#");
		}

		printf("\n");


	}



    return 0;
}
